core/css/icons.scss

```
@@ -479,7 +479,10 @@ img, object, video, button, textarea, input, select, div[contenteditable='true']
 	@include icon-color('search', 'actions', $color-black, 1, true);
 }
 
-
 .icon-talk {
 	@include icon-color('app-dark', 'spreed', $color-black, 1);
 }
+
+.nav-icon-systemtagsfilter {
+	@include icon-color('tag', 'systemtags', $color-black);
```

We can just use the icon from core here and remove the one from apps/systemtags/img/tag.svg

```suggestion
	@include icon-color('tag', 'actions', $color-black, 1, true);
```

---

Actually the class doesn't seem to be used anywhere, so maybe we can just get rid of it.

---

It is used. Generated dynamically. Inspect the tags icon in the left sidebar

---

But resolved :D